docs/install/install_instructions.md

```diff
@@ -116,22 +118,207 @@ pip3 install neuron
+Follow the installation box steps.   
+
+PS: The `pip install neuron` will not work properly in Windows. 
+
+---------------------- 
+
+#### Step 7: install any other package as matplotlib 
+In order to install any other package open a CDM or Windows PowerShell and try: 
+ 
+`conda install <name_of_the_package>`
+ 
+Example: 
+```bash
+conda install matplotlib 
+conda install plotly
+conda install panda
+```
+ 
+PS: `pip install matplotlib` may also works
+ 
+More information at (https://matplotlib.org/stable/users/getting_started/) 
+
+---------------------- 
+
+#### Step 8: Enjoy Neuron!!  
+Some neurons (cool 3D) examples [here](https://nrn.readthedocs.io/en/8.2.4/python/visualization/plotshapeclass.html).  
+
+Download files [c91662.ses](https://github.com/dohalloran/NCX_Model/blob/master/c91662.ses) and save the follow code as "example_plotly_matplotlib.py"
+
+```python
+import matplotlib
+matplotlib.use("TkAgg")  #works
+import plotly
+#from matplotlib import pyplot, cm
+
+from neuron import h, gui
+from neuron.units import mV, ms
+h.load_file('c91662.ses')
+
+'''
+#ps = h.PlotShape(False)
+#ps.variable("v")
+#ps.plot(pyplot, cmap=cm.cool)
+#pyplot.show()
+'''
+ps = h.PlotShape(False)
+ps.variable("v")
+ps.plot(plotly).show()
+```
+
+Inside the folder that your downloaded "c91662.ses" and saved "example_plotly_matplotlib.py" are, open a terminal and do:
+ 
+Open a new terminal and try: 
+```bash
+python -i example_plotly_matplotlib.py
+```
+ 
+---------------------- 
+ 
+### Common problems: 
+ 
+#### Incompatibility neuron and python version. See steps 1, 4 and 5. 
+ 
+The python version incompatibility with neuron version.
+
+---------------------- 
+```
+C:\Users\User>python3 
+Python 3.12.4 (tags/v3.12.4:8e8a4ba, Jun  6 2024, 19:30:16) [MSC v.1940 64 bit (AMD64)] on win32 
+Type "help", "copyright", "credits" or "license" for more information. 
+>>> import neuron 
+Traceback (most recent call last): 
+  File "c:\nrn\lib\python\neuron\__init__.py", line 135, in <module> 
+    from . import hoc 
+ImportError: cannot import name 'hoc' from partially initialized module 'neuron' (most likely due to a circular import) (c:\nrn\lib\python\neuron\__init__.py) 
+  
+During handling of the above exception, another exception occurred: 
+  
+Traceback (most recent call last): 
+  File "<stdin>", line 1, in <module> 
+  File "c:\nrn\lib\python\neuron\__init__.py", line 137, in <module> 
+    import neuron.hoc 
+ModuleNotFoundError: No module named 'neuron.hoc' 
+>>> 
+```
+
+---------------------- 
+
+#### "PATH" problems. See step 2. 
+ 
+A PATH problem that I'm not sure how to reproduce all error possibilities.  But it seems to be correlated with the python installation process (from Windows Microsoft Store (as an app) or by Python Webpage and/or Conda) and/or neuron/python order (I've stalled neuron before python and didn’t get this problem) 
+ 
```

Comment:
a nitpick: As this is inside README, I think it better to write this differently:

```
There can be PATH-related issues that may result in errors during the runtime.
These issues often stem from how Python is installed, whether it's through the Windows
Microsoft Store, the Python website, or via Conda. Additionally, the order of installation
(Neuron before Python, for example) can also affect the setup.
```

By the way, `stalled` -> `installed`.